spec/factories/edition_factory.rb

```diff
@@ -186,5 +187,13 @@
     trait :not_political do
       system_political { false }
     end
+
+    trait :current_government do
+      government_id { Government.current.id }
+    end
+
+    trait :past_government do
+      government_id { Government.past.first.id }
```

Comment:
By default there will be the whole set of governments that are defined in the yaml file. The way we can determine our own governments is by mocking the `Government.all` method to return whichever governments we create. So this code here works because there's going to be 56 governments in the database by default.